tests/utils/datavolume.go

```diff
@@ -44,12 +44,6 @@ const (
 
 // CreateDataVolumeFromDefinition is used by tests to create a testable Data Volume
 func CreateDataVolumeFromDefinition(clientSet *cdiclientset.Clientset, namespace string, def *cdiv1.DataVolume) (*cdiv1.DataVolume, error) {
-	if IsHostpathProvisioner() {
-		if def.ObjectMeta.Annotations == nil {
-			def.ObjectMeta.Annotations = map[string]string{}
-		}
-		AddProvisionOnNodeToAnn(def.ObjectMeta.Annotations)
```

Comment:
It looks like you don't need this helper function anymore.

Reply:
Good point, don't need the DefaultNodeName either.